routes/web.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-Route::middleware(['web', 'auth:' . config('admin-auth.defaults.guard'), 'admin'])->group(function () {
```

Comment:
@dejwCake should  `web` be also removed?

Reply:
Both middlewares has been moved to Service provider.